R/helpers.R

```diff
@@ -81,20 +85,18 @@ r_version <- function() {
 
 use_system_requirement <- function(requirement) {
   stopifnot(is_string(requirement))
-  existing_requirements <- desc::desc_get_field("SystemRequirements", default = character(), file = proj_get())
-  existing_requirements <- utils::head(strsplit(existing_requirements, ", ?"), n = 1)
 
-  if (requirement %in% existing_requirements) {
+  desc <- proj_desc()
+  existing <- desc$get_list("SystemRequirements", default = "")
```

Comment:
I'm getting the sense that maybe `$get_list()` didn't exist when we first started using desc but now it does and it's handy. Let's make sure we're stating a good minimum version for desc.

Reply:
Yeah, exactly. I've bumped the reqiured version to the latest.